ko/news/_posts/2025-12-18-ruby-4-0-0-preview3-released.md

```diff
@@ -0,0 +1,491 @@
+
+        * `Ractor::Port#receive`
+        * `Ractor::Port#send` (또는 `Ractor::Port#<<`)
+        * `Ractor::Port#close`
+        * `Ractor::Port#closed?`
+
+        그 결과, `Ractor.yield`와 `Ractor#take`가 제거되었습니다.
+
+    * Ractor의 종료를 기다리기 위해 `Ractor#join`과 `Ractor#value`가 추가되었습니다.
+      이들은 `Thread#join` 및 `Thread#value`와 유사합니다.
+
+    * `Ractor#join`을 내부적으로 구현하는 데 사용되는 저수준 인터페이스로
+      `Ractor#monitor`와 `Ractor#unmonitor`가 추가되었습니다.
+
+    * `Ractor.select`는 이제 Ractor와 Port만 허용합니다. Ractor가 주어지면,
+      Ractor가 종료될 때 반환됩니다.
+
+    * `Ractor#default_port`가 추가되었습니다. 각 `Ractor`는 `Ractor.send`, `Ractor.receive`에서
+      사용하는 기본 포트를 가집니다.
+
+    * `Ractor#close_incoming`과 `Ractor#close_outgoing`이 제거되었습니다.
+
+    * 공유 가능한 Proc 또는 lambda를 만들기 위해 `Ractor.shareable_proc`과 `Ractor.shareable_lambda`가
+      도입되었습니다. [[Feature #21550]], [[Feature #21557]]
+
+* Range
+
+    * `Range#to_set`과 `Enumerator#to_set`은 이제 끝이 없는 범위 문제를 방지하기 위해
+      크기 검사를 수행합니다. [[Bug #21654]]
+
+    * `Range#overlap?`은 이제 무한한(경계가 없는) 범위를 올바르게 처리합니다.
+      [[Bug #21185]]
+
+    * 시작이 없는 정수 범위에서 `Range#max` 동작이 수정되었습니다.
+      [[Bug #21174]] [[Bug #21175]]
+
+* Ruby
+
+    * Ruby 관련 상수를 포함하는 새로운 최상위 모듈 `Ruby`가 정의되었습니다.
+      이 모듈은 Ruby 3.4에서 예약되었으며 이제 공식적으로 정의되었습니다.
+      [[Feature #20884]]
+
+* Ruby::Box
+
+    * 정의에 대한 분리를 제공하는 새로운 (실험적) 기능입니다.
+      "Ruby Box"에 대한 자세한 내용은 [doc/language/box.md](https://github.com/ruby/ruby/blob/master/doc/language/box.md)를 참조하세요.
+      [[Feature #21311]] [[Misc #21385]]
+
+* Set
+
+    * `Set`은 이제 코어 클래스이며, 더 이상 자동 로드되는 표준 라이브러리 클래스가 아닙니다.
+      [[Feature #21216]]
+
+    * `Set#inspect`은 이제 `Set[]` 구문을 사용하여 `eval`에 적합한 문자열을 반환합니다
+      (예: `#<Set: {1, 2, 3}>` 대신 `Set[1, 2, 3]`). 이는 Array, Hash와 같은
+      다른 코어 컬렉션 클래스와 일관성을 유지합니다. [[Feature #21389]]
+
+    * `Set#to_set`과 `Enumerable#to_set`에 인자를 전달하는 것은 폐기 예정 상태가 되었습니다.
+      [[Feature #21390]]
+
+* Socket
+
+    * `Socket.tcp`와 `TCPSocket.new`는 초기 연결에 대한 타임아웃을 지정하기 위해
+      `open_timeout` 키워드 인자를 허용합니다. [[Feature #21347]]
+
+* String
+
+    * 유니코드 17.0.0 및 이모지 버전 17.0으로 업데이트합니다.
+      [[Feature #19908]][[Feature #20724]][[Feature #21275]] (Regexp에도 적용됩니다)
+
+    * `String#strip`, `strip!`, `lstrip`, `lstrip!`, `rstrip`, `rstrip!`이
+       `*selectors` 인자를 허용하도록 확장되었습니다. [[Feature #21552]]
+
+* Thread
+
+    * `Kernel#raise`와 유사한 `Thread#raise(cause:)` 인자 지원이 도입되었습니다.
+      [[Feature #21360]]
+
+* Fiber
+
+    * `Kernel#raise`와 유사한 `Fiber#raise(cause:)` 인자 지원이 도입되었습니다.
+      [[Feature #21360]]
+
+* Fiber::Scheduler
+
+    * 주어진 예외로 fiber를 인터럽트하기 위한 `Fiber::Scheduler#fiber_interrupt`가 도입되었습니다.
+      초기 사용 사례는 IO 작업이 닫힐 때 블로킹 IO 작업을 기다리는 fiber를 인터럽트하는 것입니다.
+      [[Feature #21166]]
+
+* Pathname
+
+    * Pathname이 기본 gem에서 Ruby의 코어 클래스로 승격되었습니다.
+      [[Feature #17473]]
+
+## 표준 라이브러리 변경
+
+다음 번들 gem이 기본 gem에서 승격되었습니다.
+
+* ostruct 0.6.3
+* pstore 0.2.0
+* benchmark 0.5.0
+* logger 1.7.0
+* rdoc 6.17.0
+* win32ole 1.9.2
+* irb 1.15.3
+* reline 0.6.3
+* readline 0.0.4
+* fiddle 1.1.8
```

Comment:
Some of the stdlib update versions are incorrect.  I've corrected the `en`, `ja`, and `uk` locales in #3735.  You may want to update this PR with the correct versions.